inhouse/command_handlers/match.py

```diff
@@ -119,7 +119,7 @@ async def send_channel(self, member: discord.Member, channel: discord.VoiceChann
             return ""
         except Exception as e:
             print(e)
-            return "<@" + member.id + "> join <#" + channel.id + ">\n"
+            return "<@" + str(member.id) + "> join <#" + str(channel.id) + ">\n"
```

Comment:
S T R I N G I N T E R P O L A T I O N please

Reply:
`return f"<@{member.id}> join <#{channel.id}\n"` sry im gonna continually squeaky wheel on this pattern